akka-persistence-typed/src/main/scala/akka/persistence/typed/keyvalue/scaladsl/KeyValueBehavior.scala

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2017-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.keyvalue.scaladsl
+
+import scala.annotation.tailrec
+
+import akka.actor.typed.BackoffSupervisorStrategy
+import akka.actor.typed.Behavior
+import akka.actor.typed.Signal
+import akka.actor.typed.internal.BehaviorImpl.DeferredBehavior
+import akka.actor.typed.internal.InterceptorImpl
+import akka.actor.typed.internal.LoggerClass
+import akka.actor.typed.scaladsl.ActorContext
+import akka.annotation.DoNotInherit
+import akka.persistence.typed.keyvalue.internal._
+import akka.persistence.typed.PersistenceId
+import akka.persistence.typed.SnapshotAdapter
+
+object KeyValueBehavior {
```

Comment:
this is the starting point for the public api for Scala, everything very similar to EventSourcedBehavior